### DISCUSSION TITLE
Feature request: z-axis control for the Camera

### DISCUSSION THREAD

**Discussion by shimonshomron** [keiwando/toonsquid#71] [Ideas]
Since the app has a controlled camera object - It would be great if we could control it's position on Z-axis and for achieving more dynamic / parallax / depth of field to our scenes.
** edit ** - I mean the z-index of the layers should be animatic. The camera viewpoint should reflect their (layers) depth.

**Comment by Lioncub901** [comment]
Seems cool. Be nice for a environment.

**Comment by kavlor** [comment]
The Z Axis would need to be on the layers as well as it would need to know how far each one is from the other.
 
I would be for such a feature and was only just thinking about it myself today.

**Comment by tbrainerd** [comment]
I'd also like to opt for a "camera-free" Z-depth workflow. In other words, we could animate objects with X, Y, Z motion, scale and rotation – exactly as we want them to appear. No need to reverse engineer how the "camera" would view things (much more intuitive for most 2D animators).
